packages/utilities/src/test/setRenderSpy.ts

```diff
@@ -1,10 +1,17 @@
+/* eslint-disable */
 import { ReactWrapper } from 'enzyme';
+// Since we're not using sinon in a meaningful way, the dependency has been removed, and this file
+// ought to be removed as well. But to reduce changes of breaking anyone who might be using this
+// helper (which ideally they shouldn't, since it's not a public API), we continue to load sinon
+// as an implicit dependency, since consumers of this helper will likely have it as their depenency.
+// @ts-ignore
 import * as sinon from 'sinon';
 
 /**
  * Replaces the enzyme render function with a spy that can be used to count how many
  * times the render function was called in a test.
  * @param wrapper - The wrapper around the component instance for which you want to spy upon
+ * @deprecated This package no longer uses sinon. Please copy this function locally to continue using it.
  */
 export function setRenderSpy(wrapper: ReactWrapper<{}, {}>): sinon.SinonSpy {
```

Comment:
I'd say let's just remove this. This was never officially supported from `utilities` package

Reply:
Sounds good